README.md

```diff
@@ -301,6 +301,18 @@ private void ConfigureAuthenticationTokenHandler(IServiceCollection services)
 }
 ```
 
+You will also need to provide a `DelegatingHandler` to your endpoints if you want to handle the access token logic through the `AuthenticationTokenHandler` provided.
+
+```csharp
+private static IServiceCollection AddMyEndpoint(this IServiceCollection services, IConfiguration configuration)
+{
+	return services
+		.AddEndpoint<IMyEndpoint, MyEndpointMock>(configuration, "MyEndpoint", configure: b => b
```

Comment:
This `AddEndpoint` method doesn't exists in this scope.

Reply:
It's something custom to UnoApplicationTemplate and doesn't make sense in this context.